app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -332,7 +332,13 @@ public function getItemImageUrl($item)
         /** @var Mage_Catalog_Model_Product $_product */
         $_product = $item->getProduct();
 
-        $image = $imageHelper->init($_product, 'thumbnail', $_product->getThumbnail());
+        $image = '';
+        try {
+            if ($_product->getThumbnail()) {
+                /** @var Mage_Catalog_Helper_Image $image */
+                $image = $imageHelper->init($_product, 'thumbnail', $_product->getThumbnail());
+            }
+        } catch (Exception $e) {  }
```

Comment:
can you add comment here explaining what exception we ignore here?

Reply:
This is part of code from here - https://github.com/BoltApp/bolt-magento1/pull/238/files
by @johnmccombs1 .